README.md

```diff
@@ -10,15 +10,15 @@ Gridicon renders a single svg icon based on an `icon` prop. It takes a size prop
 
 There's a gallery with all the available icons in http://automattic.github.io/gridicons/.
 
-```
-npm install gridicons --save
+```sh
+$ npm install gridicons --save
```

Comment:
Perhaps we should not include the `$` bash mark? That makes difficult to copy-paste and I guess that highlighting it is bash code would be enough?

Reply:
I agree that makes copying harder. Let's don't include it. Syntax highlighting doesn't give us much highlighting for shell commands so what about including a small comment that this is intended to be executed in a shell?